apps/src/lib/ui/accounts/ManageLinkedAccounts.jsx

```diff
@@ -57,6 +64,43 @@ export default class ManageLinkedAccounts extends React.Component {
     console.log(error.message);
   };
 
+  cannotDisconnectGoogle = () => {
+    const {isGoogleClassroomStudent} = this.props;
+    const cannotDisconnect = this.hasAuthOption(OAUTH_PROVIDERS.GOOGLE) ? isGoogleClassroomStudent : false;
```

Comment:
If there's no need to save these into a variable, I'd just return the value directly.

Reply:
good point. i'm moving all of this to redux, so i'll address this in that follow-up PR